qcp/QRegister.py

```diff
@@ -0,0 +1,43 @@
+from QMatrices import QMatrices
```

Comment:
to fix the mypy actions failing on these lines, create a top-level `mypy.ini` file, and add the following to it:
```ini
[mypy]
ignore_missing_imports = True
```
This should suppress mypy from checking for the type stubs on our code (and throwing an error when it can't find it...)

Reply:
https://github.com/Tiernan8r/quantum_computing_project/pull/19 will do this for you.